### ISSUE TITLE
Do not redeploy projects locally currently deployed at the current version and use force flag to override

### ISSUE THREAD

**Issue by george-ayris** [HuddleEng/octopose#16]
This behvaiour already exists for remote deployments, but not for local deployments, and would allow people to easily and quickly do a daily refresh on their entire system using Octopose.

**Comment by domgreen** [comment]
Agreed ... Vision was to make the remote and local as similar as possible.
👍
